components/places/src/ffi.rs

```diff
@@ -280,34 +280,8 @@ implement_into_ffi_by_delegation!(
     msg_types::BookmarkNode
 );
 
-/// Implements [`IntoFfi`] for the provided types (more than one may be passed in) implementing
-/// `uniffi::ViaFfi` (UniFFI auto-generated serialization) by delegating to that implementation.
-///
-/// This is only necessary because we have a kinda "hybrid" FFI situation -
-/// some things generated via UniFFI, others by hand. Because this macro only
-/// makes sense in this Frankenstein world it's not in UniFFI itself.
-#[macro_export]
-macro_rules! implement_into_ffi_by_uniffi {
-    ($($FFIType:ty),* $(,)*) => {$(
-        unsafe impl ffi_support::IntoFfi for $FFIType where $FFIType: uniffi::ViaFfi {
-            type Value = <Self as uniffi::ViaFfi>::FfiType;
-            #[inline]
-            fn ffi_default() -> Self::Value {
-                Default::default()
-            }
-
-            #[inline]
-            fn into_ffi_value(self) -> Self::Value {
-                <Self as uniffi::ViaFfi>::lower(self)
-            }
-        }
-    )*}
-}
-
-implement_into_ffi_by_uniffi!(crate::storage::history_metadata::HistoryMetadata);
```

Comment:
Good question! I believe it was the good work Ben did in getting rid of ViaFfi and now using the Ffi converter https://github.com/mozilla/uniffi-rs/pull/1008/ though this ADR explains well why we got rid of ViaFffi in the first place https://github.com/mozilla/uniffi-rs/blob/main/docs/adr/0006-external-types.md 

So I _think_ from what I understand we don't actually need this anymore since uniffi can now support external types more natively rather than us needing to make a wrapper outside the crate because that comment eloquently says ("Only makes sense in this Frankenstein world it's not in UniFFI itself") lol

Reply:
I'm not really sure about this one either.  I think skhamis is right, it seems like some of the work we did made it easier to integrate with the Frankenstein world.  If I had to guess maybe it was related to getting rid of the `ffi-support` dependency?